src/kwenta_v2_sdk.py

```diff
@@ -0,0 +1,660 @@
+import json
+import os
+import time
+import warnings
+import random
+import pandas as pd
+import requests
+import numpy as np
+from web3 import Web3
+from src.abi_store import *
+from src.kwenta_config import *
+
+pd.set_option('display.max_rows', None)
+pd.set_option('display.max_columns', None)
+warnings.filterwarnings('ignore')
+
+class kwenta:
+    def __init__(self, provider_rpc:str, wallet_address:str, private_key:str=None):
+        #init account variables + contracts
+        self.web3 = Web3(Web3.HTTPProvider(provider_rpc))
+        self.private_key = private_key
+        self.wallet_address = wallet_address
+        self.allmarket_listings, self.PerpsV2MarketData_abi, self.PerpsV2Market_abi, self.susd_token = self.init_markets()
+        self.token_list = list(self.allmarket_listings.keys())
+
+    def init_markets(self):
+        """
+        Initializes all market contracts
+        ...
+
+        Attributes
+        ----------
+        N/A
+        """
+        # load the PerpsV2MarketData ABI
+        with open('.\src\PerpsV2MarketData.json') as json_file:
+            PerpsV2MarketData_abi = json.load(json_file)
+        # load the PerpsV2Market ABI
+        with open('.\src\PerpsV2Market.json') as json_file:
```

Comment:
```suggestion
        with open('./src/PerpsV2Market.json') as json_file:
```
I think there's a better way to deal with this file path, since this will only work when some script is run from the root directory